app/modules/audit_logs/models.py

```diff
@@ -21,16 +18,20 @@ class AuditLog(db.Model, Timestamp):
     guid = db.Column(
         db.GUID, default=uuid.uuid4, primary_key=True
     )  # pylint: disable=invalid-name
-    module_name = db.Column(db.String(length=50), nullable=True)
+    module_name = db.Column(db.String(length=50), index=True, nullable=True)
 
     # Item and user guids intentionally not backrefs to other models as we want to maintain the log even after
     # the item has been removed
-    item_guid = db.Column(db.GUID, nullable=True)
-    user_email = db.Column(db.String(length=120), nullable=False)
+    item_guid = db.Column(db.GUID, index=True, nullable=True)
+    user_email = db.Column(db.String(), nullable=False)
```

Comment:
If you want to remove the length you can change from `db.String` to `db.Text`.  (I checked, a migration is generated)  The database type will change from `VARCHAR` to `TEXT` and I think `VARCHAR` requires a maximum length.

Reply:
Oh actually it looks like that's not true:

```
houston=# \d audit_log    
                          Table "public.audit_log"
   Column    |            Type             | Collation | Nullable | Default
-------------+-----------------------------+-----------+----------+---------
 created     | timestamp without time zone |           | not null |
 updated     | timestamp without time zone |           | not null |
 guid        | uuid                        |           | not null |
 module_name | character varying(50)       |           |          |
 item_guid   | uuid                        |           |          |
 user_email  | character varying(120)      |           | not null |
 message     | character varying(240)      |           |          |
 audit_type  | character varying           |           | not null |
Indexes:
    "pk_audit_log" PRIMARY KEY, btree (guid)

```

`audit_type` is `VARCHAR` but doesn't have a length ... :thinking:

Reply:
Gone with the drop and recreate the column idea instead. This is not a problem in this case as there will be no existing audit logs anywhere in the DBs that anyone cares about but in future we do need to know how to do this stuff.